app/src/pages/CollectionPage/CollectionPage.tsx

```diff
@@ -371,6 +372,14 @@ export const CollectionPage = (): React.ReactElement => {
                         <Text variant='small'>Average value</Text>
                       </Stack>
                     </Stack>
+                    <Stack.Item gutterAfter={PaddingSize.Narrow}>
+                      <Text variant='header3'>Recent Tweets</Text>
+                      <TwitterTimelineEmbed
+                        sourceType='profile'
+                        screenName='TokenHunt'
```

Comment:
its not our tweets, it should be the tweets of the collection, and should only show if they have a twitterUsername field